container/ClearSky/src/main/java/io/egen/clearskyboot/constants/Queries.java

```diff
@@ -0,0 +1,28 @@
+package io.egen.clearskyboot.constants;
+
+public final class Queries {
```

Comment:
I would like to have the queries on the same place where you define entities because
1. it avoids having long classes like this.
2. When you read the query defined with the entity., you know what is the context.